packages/jest-cli/src/reporters/DefaultTestReporter.js

```diff
@@ -19,14 +19,15 @@ const FAIL = chalk.reset.bold.bgRed(' FAIL ');
 const PASS = chalk.reset.bold.bgGreen(' PASS ');
 
 const FAIL_COLOR = chalk.bold.red;
+const LONG_TEST_COLOR = chalk.reset.bold.bgRed;
 const PASS_COLOR = chalk.bold.green;
 const PENDING_COLOR = chalk.bold.yellow;
 const RUNNING_TEST_COLOR = chalk.bold.gray;
+const SNAPSHOT_SUMMARY = chalk.bold.green;
 const TEST_NAME_COLOR = chalk.bold;
-const LONG_TEST_COLOR = chalk.reset.bold.bgRed;
 
 const print = (word, count) => `${count} ${word}${count === 1 ? '' : 's'}`;
-
+const pluralizeSnapshot = num => 'snapshot' + (num > 1 ? 's' : '');
```

Comment:
wait, why can't we use `print` from above? `print('snapshot', count) does the same? (Also, we should rename `print` to `pluralize` :)